lib/Agrammon/TechnicalParser.pm6

```diff
@@ -26,3 +27,13 @@ sub parse-technical(Str $to-parse) is export {
         die "Failed to parse technical configuration: unknown error";
     }
 }
+
+sub load-technical(IO::Path $model-path, Str $technical-file) is export {
+    my $tech-input = $model-path.IO.add($technical-file // 'technical.cfg');
```

Comment:
Note that `add` will interpret any `..` sequences, so there's a potential vulnerability here given we're letting the caller of the API pick the path.

Reply:
Fixed